CHANGELOG.md

```diff
@@ -45,6 +45,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Whirlwind](https://github.com/Whirlwind)
   [#7878](https://github.com/CocoaPods/CocoaPods/pull/7878)
 
+* Add a `--deployment` flag to `pod install` that errors if there are any
+  changes to the Podfile or Lockfile.  
+  [Samuel Giddins](https://github.com/segiddins)
```

Comment:
The description sounds like its saying that it will error if the command itself changes the Podfile or Lockfile.

Maybe "... that errors if the Podfile and Lockfile are not in sync" or "... if `pod install` would trigger an update to the Lockfile"

Reply:
I dont think it's about "sync" though, its about them being dirty unless thats what you meant with "sync" on an individual file basis

Reply:
hmm yeah. words are hard